linux/fedora-rpm/audacity.spec

```diff
@@ -0,0 +1,245 @@
+# Spec file is based on https://src.fedoraproject.org/fork/imcinerney/rpms/audacity/tree/im/depupdate
+# Compile options:
+# invoke with: rpmbuild --with ffmpeg --with local_ffmpeg audacity.spec to use local ffmpeg
+%bcond_with     ffmpeg
+%bcond_with     local_ffmpeg
+
+#global commit0 53a5c930a4b5b053ab06a8b975458fc51cf41f6c
+#global shortcommit0 #(c=#{commit0}; echo ${c:0:7})
+
+Name: audacity
+
+Version: 3.0.3
```

Comment:
Why are we building an RPM package rather than leaving it to distros? (Just curious, I've nothing against building distro packages.)

Reply:
I actually plan to add Debian and Arch here as well, so we can regularly check that everything builds as expected.